packages/app-store/_utils/useAddAppMutation.ts

```diff
@@ -20,6 +28,13 @@ function useAddAppMutation(_type: App["type"] | null, options?: Parameters<typeo
       isOmniInstall = variables.isOmniInstall;
       type = variables.type;
     }
+
+    //Type _other_calendar apps aren't supported yet with CLI properly.
+    // If we change their type to the same as slug the app logic that depends on checking the presence of _calendar breaks and if we keep the type as is, it is different from slug.
+    // TODO: Work on removing dependency of having type contain _calendar
```

Comment:
There are more than a few apps with this characteristic. I would prefer to tackle them together in a future fix than having to tackle just one introducing logic to handle just one exception. Thoughts?

Reply:
Yeah, those are old apps that are not created using CLI. It should be fixed for all of those together, till then we can make a whitelist here.